### ISSUE TITLE
DEM

### ISSUE THREAD

**Issue by shelbysawyer** [Riverscapes/RiverscapesXML#285]

**Comment by joewheaton** [comment]
@shelbysawyer, I suspect this is now fixed with all the latest changes @MattReimer has made to WebRAVE. [Here's one](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/e9e4e957-ba92-47cb-b190-12cf4521f0e5) I was just poking around in:
![image](https://user-images.githubusercontent.com/9723905/137572702-d3450f21-4021-47d0-b591-81d2bb621d5c.png)

**Comment by shelbysawyer** [comment]
@joewheaton You're right-- I have a video I need to post, and this issue is addressed in it. Please leave this open for now and I'll close with the video when I upload.
